java-engine/src/test/java/io/getunleash/engine/UnleashEngineTest.java

```diff
@@ -44,6 +49,43 @@ public static String loadFeaturesFromFile(String filePath) {
         }
     }
 
+    static IStrategy alwaysTrue =
```

Comment:
Wow, I've been away from Java a long time, I didn't even know you could do this now!